geomstats/geometry/euclidean.py

```diff
@@ -63,6 +63,40 @@ def exp(self, tangent_vec, base_point):
         """
         return tangent_vec + base_point
 
+    @property
+    def is_connected(self):
+        """Indicates whether the geometric object is connected.
+
+        Returns
+        -------
+        bool
```

Comment:
e.g. `is_connected : bool` for consistency with other docstrings (though this style probably makes more sense).